Makefile

```diff
@@ -45,17 +45,19 @@ ifneq (${EXTRA_ENV_FILE},'')
     export
 endif
 
+# when we don't want shared services to be used, we don't need external networks
+COMPOSE_FILE_BARE=$(shell echo '${COMPOSE_FILE}'|sed -e 's|;\?docker/run.yml||')
 
 HOSTS=127.0.0.1 world.productopener.localhost fr.productopener.localhost static.productopener.localhost ssl-api.productopener.localhost fr-en.productopener.localhost
 # commands aliases
 DOCKER_COMPOSE=docker compose --env-file=${ENV_FILE} ${LOAD_EXTRA_ENV_FILE}
-# Ensure shared_network is referenced when running locally
-DOCKER_COMPOSE_RUN=COMPOSE_FILE="${COMPOSE_FILE};docker/run.yml" ${DOCKER_COMPOSE}
+# docker command that do not need the shared network
+DOCKER_COMPOSE_BARE=COMPOSE_FILE="${COMPOSE_FILE_BARE}" ${DOCKER_COMPOSE}
 # we run tests in a specific project name to be separated from dev instances
 # keep web-default for web contents
 # we also publish mongodb on a separate port to avoid conflicts
 # we also enable the possibility to fake services in po_test_runner
-DOCKER_COMPOSE_TEST=WEB_RESOURCES_PATH=./web-default ROBOTOFF_URL="http://backend:8881/" GOOGLE_CLOUD_VISION_API_URL="http://backend:8881/" COMPOSE_PROJECT_NAME=${COMPOSE_PROJECT_NAME}_test COMPOSE_FILE="${COMPOSE_FILE};${DEPS_DIR}/openfoodfacts-shared-services/docker-compose.yml" PO_COMMON_PREFIX=test_ MONGO_EXPOSE_PORT=27027 MONGODB_CACHE_SIZE=4 ODOO_CRM_URL= docker compose --env-file=${ENV_FILE}
+DOCKER_COMPOSE_TEST=COMMON_NET_NAME=${COMMON_NET_NAME}_test WEB_RESOURCES_PATH=./web-default ROBOTOFF_URL="http://backend:8881/" GOOGLE_CLOUD_VISION_API_URL="http://backend:8881/" COMPOSE_PROJECT_NAME=${COMPOSE_PROJECT_NAME}_test COMPOSE_FILE="${COMPOSE_FILE};${DEPS_DIR}/openfoodfacts-shared-services/docker-compose.yml" PO_COMMON_PREFIX=test_ MONGO_EXPOSE_PORT=27027 MONGODB_CACHE_SIZE=4 ODOO_CRM_URL= docker compose --env-file=${ENV_FILE}
```

Comment:
Should maybe suffix the COMMON_NET_NAME with po_test rather than just test as we want these resources to be isolated to the PO project

Reply:
Although strictly speaking none of the compose files being referenced for test purposes should mention the COMMON_NET_NAME

Reply:
I don't understand what you mean !